components/family-studies/ui/src/main/resources/PhenoTips/FamilySheet.xml

```diff
@@ -97,33 +97,11 @@
 ##
 ## Extensions displayed after the content
 #foreach ($e in $services.uix.getExtensions('phenotips.familyRecord.after', {'sortByParameter' : 'order'}))
-  #set ($extension_disabled = ($e.getParameters().get('enabled') == 'false'))
+#set ($extension_disabled = ($e.getParameters().get('enabled') == 'false'))
   #if (!$extension_disabled)
     {{html clean=false}}$services.rendering.render($e.execute(), 'xhtml/1.0'){{/html}}
 
   #end
 #end
-#end
-
-#foreach ($e in $services.uix.getExtensions('org.phenotips.familySheet'))
-  #set ($extension_disabled = ($e.getParameters().get('disabled') == 'true'))
-  #if (!$extension_disabled)
-    {{html wiki=false clean=false}}$services.rendering.render($e.execute(), 'xhtml/1.0'){{/html}}
-
-  #end
-#end
-{{/velocity}}
-
-{{comment}}{{velocity}}
-  #foreach ($e in $services.uix.getExtensions('org.phenotips.patientSheet.section.family-info'))
-    #if ($e.getId() == 'org.phenotips.patientSheet.field.pedigree')
-      #set ($pedigreeHtml = $services.rendering.render($e.execute(), 'xhtml/1.0'))
-    #end
-  #end
-  #if ("$!{pedigreeHtml}" != '')
-    (% id='pedigree' class='chapter clinical-info pedigree-family-section' %)(((
-      {{html wiki=false clean=false}}$pedigreeHtml{{/html}}
-    )))
-  #end
-{{/velocity}}{{/comment}}</content>
+#end</content>
```

Comment:
Technically, closing the macro at the end of the content is not mandatory. But yes, there should be a closing tag, thanks for noticing.